met-web/src/components/layout/SideNav/SideNav.tsx

```diff
@@ -100,14 +91,25 @@ const DrawerBox = ({ isMediumScreen, setOpen }: DrawerBoxProps) => {
                             '&:hover, &:active, &:focus': {
                                 backgroundColor: 'transparent',
                             },
+                            padding: 2,
+                            pl: 4,
                         }}
                         data-testid={`SideNav/${route.name}-button`}
                         onClick={() => {
                             navigate(route.path);
                             setOpen(false);
                         }}
                     >
-                        {handleFontAwesomeIcon(route, itemType)}
+                        <FontAwesomeIcon
+                            style={{
+                                fontSize: '1.1rem',
+                                color: 'selected' === itemType ? Palette.primary.light : Palette.text.primary,
+                                paddingRight: '0.75rem',
+                                width: '1.1rem',
+                            }}
+                            icon={route.icon ?? faLinkSlash}
```

Comment:
The double question mark paired with the integration of the icons with the route makes for a less verbose solution.